README.md

```diff
@@ -1,8 +1,24 @@
-![Maven Build](https://github.com/osscameroon/js-generator/actions/workflows/maven.yml/badge.svg)
```

Comment:
Please, don't remove the badges, it gives some useful information.

Reply:
Ok

Reply:
I just looked again at your work then I saw that you didn't delete the badges, sorry for the inconvenience. I will do some corrections in order to merge it this weekend, there some good stuff 💯

Reply:
Please, keep this style, just add contributors at MIT license

```
![Maven Build](https://github.com/osscameroon/js-generator/actions/workflows/maven.yml/badge.svg)
[![codecov](https://codecov.io/gh/osscameroon/js-generator/branch/main/graph/badge.svg?token=QJEBIRY8JJ)](https://codecov.io/gh/osscameroon/js-generator)
[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```